docs/layouts/shortcodes/generated/rocksdb_native_metric_configuration.html

```diff
@@ -26,6 +26,18 @@
             <td>Boolean</td>
             <td>Monitor block cache capacity.</td>
         </tr>
+        <tr>
```

Comment:
We may also need to add some descriptions about the new statistics in the "RocksDB Native Metrics" module of "config.md".
As you could see current content: "The metrics here are scoped to the operators and then further broken down by column family;".

Reply:
Thanks for the reminder, and I will update the `config.md`.